Dockerfile

```diff
@@ -0,0 +1,14 @@
+# you can use this container for running your tests
+# this is a distribution mechanism of conu
+
+FROM registry.fedoraproject.org/fedora:26
+
+# the tests are being expected here
+VOLUME /tests
+WORKDIR /tests
+CMD ["pytest-3", "-v", "/tests"]
+
+RUN dnf install -y acl python3-pyxattr python3-pytest python3-docker python3-six docker-latest docker atomic && dnf clean all
```

Comment:
why to install just `python3-pytest` and not `python2-pytest` it is not mandatory for conu to install pytest anyhow. It is test frametowrk (used by internal testsuite)

But it is good to have these test requirements somewhere mentioned.